kern/src/syscall.c

```diff
@@ -697,8 +697,6 @@ static ssize_t sys_fork(env_t* e)
 	assert(env != NULL);
 	proc_set_progname(env, e->progname);
 
-	env->heap_top = e->heap_top;
-	env->ppid = e->pid;
```

Comment:
I was able to remove this line because it is now done inside proc_alloc() automatically.